app.js

```diff
@@ -62,29 +29,69 @@ var getCookiesPerHour = function(objectName){
   }
 };
 
+CookieStore.prototype.makeTheTable = function(tableToDrawInto){
+  var hourlySalesRowEl = document.createElement('tr');
+  console.log(hourlySalesRowEl);
+  var storeNameEl = document.createElement('th');
+  console.log(storeNameEl);
+  storeNameEl.textContent = this.name;
+  hourlySalesRowEl.appendChild(storeNameEl);
+
+  for(var hours = 0; hours < this.cookiesPerHour.length; hours++){
+    var hourlyCookieSales = document.createElement('td');
+    console.log(hourlyCookieSales);
+    hourlyCookieSales.textContent = this.cookiesPerHour[hours];
+    console.log(hourlyCookieSales);
+    hourlySalesRowEl.appendChild(hourlyCookieSales);
+  }
+  tableToDrawInto.appendChild(hourlySalesRowEl);
+};
+
 //Call a function for each city to populate the arrays in each object.
 getCookiesPerHour(firstAndPike);
 getCookiesPerHour(seatacAirport);
 getCookiesPerHour(seattleCenter);
 getCookiesPerHour(capitolHill);
 getCookiesPerHour(alki);
 
-//Create function to create an ul and insert the array into it.
-function putSalesProjections(location){
-  console.log(location.name + '-projections');
-  var ulElement = document.getElementById(location.name + '-projections');
+var tableEl = document.createElement('table');
```

Comment:
You should think about breaking this out into a function. That way, you can group the function call down at the bottom, and just end up with something like:

```javascript
drawLabelsRow();
drawEachStoreRow();
drawTotalsRow();
```

or, you could even break that out into a function like this:

```javascript
function drawSalesTable() {
  drawLabelsRow();
  drawEachStoreRow();
  drawTotalsRow();
}
```
and then just call drawSalesTable as the last line.